components/edit-collective/sections/InvoicesReceipts.js

```diff
@@ -58,7 +58,7 @@ const InvoicesReceipts = ({ collective }) => {
       </P>
       {error && (
         <MessageBox type="error" fontSize="14px" withIcon mb={3}>
-          {getErrorFromGraphqlException(setValue).message}
+          {i18nGraphqlException(intl, setValue)}
```

Comment:
This one seems wrong, it was so before this PR. Let's fix it while we're at it.

```suggestion
          {i18nGraphqlException(intl, error)}
```